aten/src/ATen/native/native_functions.yaml

```diff
@@ -13902,55 +13902,55 @@
   autogen: _scaled_dot_product_attention.out
 
 # This aten function is kept so that we can test the choice function from Python
-- func: _fused_sdp_choice(Tensor query, Tensor key, Tensor value, Tensor? attn_mask=None, float dropout_p=0.0, bool is_causal=False) -> int
+- func: _fused_sdp_choice(Tensor query, Tensor key, Tensor value, Tensor? attn_mask=None, float dropout_p=0.0, bool is_causal=False, float? scale=None) -> int
```

Comment:
nit: Did you mean to add the `, *,`?

Reply:
Didn't add in flash_atten_forward and backward since meant to really be c++ funcs where there isn't any kwarg support, but should add here

Reply:
ehh for consistency I'll add everywhere